lang/java/BUILD.md

```diff
@@ -44,6 +46,12 @@ To perform "fast" build, which skips tests and all code quality enforcement tool
 
     mvn clean install -DskipTests -TC1 -P!code-quality
 
+
+  * Please follow the following wiki for all running all java builds [Java Build Wiki](https://cwiki.apache.org/confluence/display/REEF/Linux), the concept of ulimit increase allows more file descriptors to be opened for some of the unit tests in REEF
+
+  * The builds for REEF have been tested on Ubuntu versions 16.04 and 16.10, Windows Server 2016
```

Comment:
You might want to poll the dev list to see which platforms everyone has tested on. For sure it has been tested on Windows 10 and possibly some other Linux distributions.